### ISSUE TITLE
Neurocarta background job is broken without DO

### ISSUE THREAD

**Issue by arteymix** [PavlidisLab/Gemma#706]
I think we should consider loading the disease ontology but not adding it to the `OntologyService`.

```
[Gemma - schedulerFactoryBean_Worker-10 (2023-06-01 02:31:57,238)] ERROR org.quartz.core.JobRunShell.run(232) | Job DEFAULT.neurocartaJobDetail threw an unhandled Exception:
JobMethodInvocationFailedException: Invocation of method 'writeAllEvidenceToFile' on target class [class com.sun.proxy.$Proxy426] failed; nested exception is java.lang.RuntimeException: Ontologies are not fully loaded yet, try again soon
org.springframework.scheduling.quartz.JobMethodInvocationFailedException: Invocation of method 'writeAllEvidenceToFile' on target class [class com.sun.proxy.$Proxy426] failed; nested exception is java.lang.RuntimeException: Ontologies are not fully loaded yet, try again soon
    at org.springframework.scheduling.quartz.MethodInvokingJobDetailFactoryBean$MethodInvokingJob.executeInternal(MethodInvokingJobDetailFactoryBean.java:321)
    at org.springframework.scheduling.quartz.QuartzJobBean.execute(QuartzJobBean.java:114)
    at org.quartz.core.JobRunShell.run(JobRunShell.java:223)
    at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:549)
```

**Comment by arteymix** [comment]
Fixed in 258eb73cfa2a78d73945ff4168160a8904db95c7, I simply removed the scheduled job.

**Comment by arteymix** [comment]
It's also in 1.30 in ec6c6e06da3715713144bd12165cb745f5cf8c52.
